CONTRIBUTING.rst

```diff
@@ -102,7 +102,7 @@ Sign off
 In order to have a concrete record that your contribution is intentional
 and you agree to license it under the same terms as the project's license, we've adopted the
 same lightweight approach that the Linux Kernel
-(https://www.kernel.org/doc/Documentation/SubmittingPatches), Docker
+(https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin), Docker
```

Comment:
could you give the link some text that doesn't take up a whole line in the doc?

Reply:
Sure thing - i think i have got the right RST formatting when doing that.